azerothcore/azerothcore-wotlk
# fix(DB/creature_loot_template): Elixir of Demonslaying should not drop

## Changes Proposed:
-  Deletes Elixir of Demonslaying drop from 7 mobs, that shouldn't drop the elixir

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7986
- Closes https://github.com/chromiecraft/chromiecraft/issues/1786

## SOURCE:

https://tbc.wowhead.com/item=9224/elixir-of-demonslaying
https://github.com/azerothcore/azerothcore-wotlk/issues/7744

## Tests Performed:

- Ran SQL, no errors
- checked db

## How to Test the Changes:


1. check db with this query:

select creature_template.name, creature_loot_template.entry, creature_loot_template.chance from creature_loot_template join creature_template on creature_loot_template.Entry=creature_template.lootid where creature_loot_template.item="9224";

2. should not list a creature

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
